### ISSUE TITLE
Missing severity level while importing Netsparker report

### ISSUE THREAD

**Issue by miynat** [DefectDojo/django-DefectDojo#1058]
I've deployed Defect Dojo on my machine with Docker.

(DefectDojo version -> 2b5d417: Update README.md [2019-04-12 14:49:15 -0500])

Netsparker has a new severity level called *Best Practice*.

When I try to import a Netsparker report with this severity level I get an HTTP 500 error on the browser, and that traceback message is logged into the console:

uwsgi_1         | item blowup
uwsgi_1         | Subresource Integrity (SRI) Not Implemented
uwsgi_1         | Internal Server Error: /engagement/2/import_scan_results
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
uwsgi_1         |     response = get_response(request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
uwsgi_1         |     response = self._get_response(request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
uwsgi_1         |     response = self.process_exception_by_middleware(e, request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
uwsgi_1         |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
uwsgi_1         |     return view_func(request, *args, **kwargs)
uwsgi_1         |   File "./dojo/engagement/views.py", line 531, in import_scan_results
uwsgi_1         |     if Finding.SEVERITIES[sev] > Finding.SEVERITIES[min_sev]:
uwsgi_1         | KeyError: u'Bestpractice'

Here you can find an example report to reproduce that behaviour:

[php.testsparker.com - Vulnerabilities List (JSON).json.zip](https://github.com/DefectDojo/django-DefectDojo/files/3088737/php.testsparker.com.-.Vulnerabilities.List.JSON.json.zip)

Please let me know if you need any further details from my end.

**Comment by dr3dd589** [comment]
hello, @devGregA So, Here Netsparker uses a new category  name for severity like:
```
	"Name": "Missing X-XSS-Protection Header",
	"Severity": "BestPractice",
```   
and 
```
	"Name": "Content Security Policy (CSP) Not Implemented",
	"Severity": "BestPractice",
```

It is ok to take it in low severity category?

**Comment by rasinfosec** [comment]
I can add make a new change / pull request to account for the new BestPractice.  

I would probably just do the below and the severity for *BestPractice* and *Information* in the Json file would become *Info*.    

       if sev == "Low":
           sev = "Low"
       elif sev == "Medium":
           sev = "Medium"
       elif sev == "High":
           sev = "High"
       elif sev == "Critical":
           sev == "Critical"
       else:
           sev = "Info"

**Comment by dr3dd589** [comment]
yeah, I also think that's a good approach. :)

**Comment by valentijnscholten** [comment]
Fixed by #1067, let us know if it doesn't work.
